### PR TITLE
Add required inputs into default configuration

### LINKED ISSUE #30: Support freezed and drift dependencies by default in build.yaml
Freezed and Drift are often used in mappr. Now users need to tweak build.yaml for correct order of generators. 

With this change, we can support freezed and Drift by default.

### PR DESCRIPTION
**Pull request by petrnymsa**
Fixes #30

### CODE REVIEW

File: packages/auto_mappr/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## Unpublished
+- Add `required_inputs` with  `.freezed.dart` and `.drift.dart` by default
```
Comment (tenhobi) [inline packages/auto_mappr/CHANGELOG.md:2]:
```suggestion
- Add `required_inputs` with `.freezed.dart` and `.drift.dart` by default.
```

File: packages/auto_mappr/example/lib/freezed_example.dart [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  final String email;
+  final String loginIdentifier;
+  final DateTime updatedAt;
+
+  UserAbstract({
+    required this.email,
+    required this.loginIdentifier,
+    required this.updatedAt,
+  });
+}
```
Comment (tenhobi) [inline packages/auto_mappr/example/lib/freezed_example.dart:28]: Is this used somewhere?

**Review by tenhobi — CHANGES_REQUESTED** [review]

File: packages/auto_mappr/example/lib/freezed_example.dart (hunk 2 above)
Comment (petrnymsa) [reply packages/auto_mappr/example/lib/freezed_example.dart]: Removed

File: packages/auto_mappr/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,6 @@
+## Unpublished
```
Comment (tenhobi) [inline packages/auto_mappr/CHANGELOG.md:1]:
```suggestion
## Unreleased
```

**Review by tenhobi — CHANGES_REQUESTED** [review]

File: packages/auto_mappr/pubspec.yaml [hunk 4]
```diff
@@ -23,8 +23,8 @@ dependencies:
 
 dev_dependencies:
   build_runner: ^2.0.0
-  dart_code_metrics: ^5.7.0
-  generator_test: ^0.2.0
+  dart_code_metrics: ^5.7.0 
+  generator_test: ^0.2.0 
   json_annotation: ^4.8.0
   json_serializable: ^6.6.1
```
[the remark this answers is not in the document]
Comment (petrnymsa) [reply packages/auto_mappr/pubspec.yaml:29]: I think tests should stay in generator package

File: melos.yaml [hunk 5]
```diff
@@ -2,7 +2,7 @@ name: auto_mappr_workspace
 
 packages:
   - packages/**
-  - packages/**/example
+  - examples/**
```
Comment (tenhobi) [inline melos.yaml:5]: IMO we need both `examples/**` and `packages/**/example`.

**Review by tenhobi — APPROVED** [review]
jiffy

**Pull request merged**